### PR TITLE
Add load test for log module

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
## Purpose
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/2366
## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: .github/workflows/process-load-test-result.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        env:
+          GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
+      - name: Approve PR
+        if: ${{ github.event.client_payload.status == 'success' }}
+        run: gh pr review --approve
+        env:
+          GITHUB_TOKEN: ${{ secrets.BALLERINA_REVIEWER_BOT_TOKEN }}
+      - name: Merge PR
+        if: ${{ github.event.client_payload.status == 'success' }}
+        run: gh pr merge --auto --merge --delete-branch
```
Comment (dilanSachi) [inline .github/workflows/process-load-test-result.yml:39]: This is an old version of the yml file. Correct one can be found in https://github.com/ballerina-platform/module-ballerinax-kafka/blob/master/.github/workflows/process-load-test-result.yml

File: .github/workflows/trigger-load-tests.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        description: >
+          List of test names. This needs to be filled only if you want to run a specific set of tests. Example: foo,bar
+        required: false
+      clusterName:
+        description: 'Cluster name'
+        default: 'log-perf-cluster-test'
+        required: false
+      dispatchType:
+        description: >
+          Repository dispatch type. Specify the dispatch type in order to trigger the result processing workflow.
```
Comment (dilanSachi) [inline .github/workflows/trigger-load-tests.yml:16]: here also. https://github.com/ballerina-platform/module-ballerinax-kafka/blob/master/.github/workflows/trigger-load-tests.yml

File: .github/workflows/process-load-test-result.yml (hunk 1 above)
Comment (MadhukaHarith92) [reply .github/workflows/process-load-test-result.yml]: Updated in a6003f3

File: .github/workflows/trigger-load-tests.yml (hunk 2 above)
Comment (MadhukaHarith92) [reply .github/workflows/trigger-load-tests.yml]: Updated in a6003f3

**Review by dilanSachi — APPROVED** [review]
LGTM

File: load-tests/log/src/app.bal [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/log;
+import ballerina/http;
+
+service http:Service /BalPerformance on new http:Listener(9090) {
```
Comment (daneshk) [inline load-tests/log/src/app.bal:20]:
```suggestion
service /load_test on new http:Listener(9090) {
```

File: load-tests/log/src/app.bal [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+    resource function get grpc() returns boolean {
```
Comment (daneshk) [inline load-tests/log/src/app.bal:22]:
```suggestion
    resource function get log() returns boolean {
```

**Review by daneshk — APPROVED** [review]

**Pull request merged**